Travelling_Salesman_Problem/Travelling_Salesman.cpp

```diff
@@ -0,0 +1,125 @@
+#include<iostream>
+using namespace std;
+ 
+int ary[10][10],completed[10],n,cost=0;
+ 
+void takeInput()
+{
+	int i,j;
+ 
+	cout<<"Enter the number of villages: ";
```

Comment:
Add Spacing around `>>`,  `<<`,  after ``for``, ``if``, around operators (+, -, *, /, <=, >=, != etc).